tests/cases/compiler/typedArrayConstructorOverloads.ts

```diff
@@ -0,0 +1,13 @@
+// @target: esnext
+// @noEmit: true
+// @noTypesAndSymbols: true
+
+// https://github.com/microsoft/TypeScript/issues/60367
+
+type TypedArrayConstructor =
+    | Float64ArrayConstructor
+    | BigInt64ArrayConstructor
+
+export function makeTypedArray(buffer: ArrayBuffer, ctr: TypedArrayConstructor) {

```

Comment:
NOTE: There is no output because this is only check for an error. Without the above changes to es2020.bigint.d.ts, an errors.txt is generated.